selendroid-common/src/main/java/io/selendroid/common/SelendroidCapabilities.java

```diff
@@ -58,6 +61,7 @@
   public static final String LAUNCH_ACTIVITY = "launchActivity";
   public static final String SELENDROID_EXTENSIONS = "selendroidExtensions";
   public static final String BOOTSTRAP_CLASS_NAMES = "bootstrapClassNames";
+  public static final String EXTRA_ARGS = "extraArgs";
```

Comment:
Perhaps it would be more clear if we call this something like `extraAUTArgs`? It isn't directly clear to me where the args go.